index.html

```diff
@@ -5673,13 +5633,17 @@ <h3 id="Scoundrel">Scoundrel</h3>
             <tr>
                <td>10
                </td>
-               <td>-
+               <td>Paralysing Strike
                </td>
-               <td>
-                  -
-               <td>-
+               <td><strong>Coat your weapon with a paralysing toxin and attack.</strong>
+                  <p>
+                     Make a weapon attack against a creature. If that creature has 0 FORT points
+                     remaining, it becomes <a class="condition" data-tooltip-container>paralysed</a>.
+                     The <a class="condition" data-tooltip-container>paralysis</a> cannot be resisted. 
```

Comment:
Maybe this part isn't necessary, as it definitely can be 'resisted' if the creature is immune